modelon/impact/client/client.py

```diff
@@ -105,6 +105,13 @@ def __init__(
 
         self._sal.add_login_retry_with(api_key)
 
+        resp = self._sal.users.get_me()
+        if 'license' not in resp['data']:
+            raise exceptions.NoAssignedLicenseError(
+                "No assigned license during login. Either out of floating "
```

Comment:
Maybe "sharable license" is the established term? At least that's what's used in the ops manual

Reply:
We might decide to change the naming in the ops manual. But we should at least change this error message to use: "<floating|sharable> Deployment Add-on licenses" to be more precise.

Reply:
After discussion we went with floating. I also have made a commit where the type of the license is more precise.